lms/static/scripts/frontend_apps/components/GradingControls.tsx

```diff
@@ -52,7 +37,7 @@ export default function GradingToolbar({
       }
     }
     // Make a copy
-    const students_ = [...unorderedStudents];
+    const students_ = unorderedStudents ? [...unorderedStudents] : [];
 
     students_.sort((student1, student2) => {
       return compareNames(student1.displayName, student2.displayName);
```

Comment:
It is outside the scope of this PR, but we could use the [Intl.Collator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) API to do case insensitive comparisons of display names instead of writing our own string-insensitive comparison. That could also handle things like accents better.